transcom/mymove
# B 21548/21547 update service items int

### [B-21548](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1022468) /  [B-21547](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1022467)

## Summary

This PR adds 4 pre approved service items to international shipments based on if they're oconus or not.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Turn on `FEATURE_FLAG_ENABLE_ALASKA=true` in .envrc file
2. Run `direnv allow` before running servers
3. Login as a TOO officer
4. Add an international HHG shipment to any move that service counseling is completed.*
5. Approve and send move
6. Confirm approval preview modal shows 4 service items
7. Confirm POE service item is shown if shipment is Oconus; otherwise POD.**
8. Click send on the move
9. Confirm All service items were added to the move and pre approved
10. Confirm you can reject/re-accept the service items
11. Confirm that the correct values are showing in the db as well
Note: There are only 4 service items because POE vs POD only shows if the move has an conus destination.

*An example would be  "600 Richardson Dr, Anchorage, AK 99505". There will be an i in front of the shipment if successful like iHHG.
** This is determined by the pickup/destination address "isOconus" field. I would just chang one of them to true for testing.

### Frontend


### Backend


## Screenshots
_**Preview international service items**_
<img width="1041" alt="Screenshot 2024-11-13 at 9 37 51 AM" src="https://github.com/user-attachments/assets/897ea51b-2235-47e3-a908-25b3a7bba23c">

_**Officer approve/reject pre approved service items**_
<img width="831" alt="Screenshot 2024-11-13 at 9 38 43 AM" src="https://github.com/user-attachments/assets/48f80692-a231-4d0e-a90b-4b73d2e00578">
